README.md

```diff
@@ -28,16 +30,68 @@ Vite:
 npm install --save-dev vite-plugin-gltf
 ```
 
-## Examples
+### 2. Minimal Configuration
 
-The plugin is unopinionated about which optimizations should be applied to your glTF models. Many optimizations are available off the shelf from [`@gltf-transform/functions`](https://gltf-transform.donmccurdy.com/functions.html):
+The plugin is unopinionated about which optimizations should be applied to your glTF models. Here is a minimal configuration that simply applies Draco compression of your assets:
 
 ```js
 // *.config.js
-import gltf from 'rollup-plugin-gltf'; // (a) Rollup
-import gltf from 'vite-plugin-gltf'; // (b) Vite
+import gltf from "rollup-plugin-gltf"; // (a) Rollup
+import gltf from "vite-plugin-gltf"; // (b) Vite
+
+import { draco } from "@gltf-transform/functions";
+
+return {
+  // ...
+  plugins: [
+    gltf({
+      transforms: [draco()],
+    }),
+  ],
+};
+```
+
+See [advanced configuration](#advanced-configuration) for more complex examples.
+
+_SvelteKit:_ it seems that in SvelteKit SSR should be disabled in order to avoid [mysterious `ReferenceError` issues](https://github.com/nytimes/rd-bundler-3d-plugins/issues/14#issuecomment-1410620469).
```

Comment:
Would you mind filing an issue (on this repo) about SvelteKit SSR too? Does seem very weird that our plugin would affect that. 🫤

Reply:
I created the issues and referenced them in the README:

https://github.com/sveltejs/kit/issues/9000
https://github.com/nytimes/rd-bundler-3d-plugins/issues/19